src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -304,12 +312,30 @@ public LoginPanel(Login myFrame) {
 			add(body, BorderLayout.NORTH);
 			add(buttons, BorderLayout.SOUTH);
 
+			if (verifyIfResourceExists(DEFAULT_CREDENTIALS_PROPERTIES)) {
+				String tooltip = FileTools.readFileToStringLineByLine(DEFAULT_CREDENTIALS_PROPERTIES, true);
+				JLabel infoButton = new JLabelInfo(new ImageIcon("rsc/icons/info_button.png"), tooltip, Color.white);
+
+				JPanel infoPanel = new JPanel();
+				infoPanel.setLayout(new FlowLayout());
+				infoPanel.add(infoButton, BorderLayout.CENTER);
+				add(infoPanel, BorderLayout.WEST);
+			}
+
 			submit.addActionListener(myFrame);
 			submit.setName("submit");
 			submit.addKeyListener(myFrame);
 			cancel.addActionListener(myFrame);
 			cancel.setName("cancel");
 			cancel.addKeyListener(myFrame);
+
 		}
 	}
+
+	private boolean verifyIfResourceExists(String path) {
```

Comment:
Minor point but this method can be made static reducing memory usage slightly.

Reply:
Yes, I'm moving it to core/utils and make it public static, thanks